### ISSUE TITLE
Add validation set monitoring

### ISSUE THREAD

**Issue by dimiz51** [dimiz51/PetsArtistGAN#2]
Add a custom test_set step to monitor validation loss as now it does not seem
to be tracked when training.
